src/System.IO.FileSystem/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,7 @@
+Compat issues with assembly System.IO.FileSystem:
+CannotMakeMemberAbstract : Member 'System.IO.FileSystemInfo.Exists' is abstract in the implementation but is not abstract in the contract.
```

Comment:
@JeremyKuhne Can you please verify that these differences in the are intentional?